src/main/kotlin/frc/robot/subsystems/roller/RollerConstants.kt

```diff
@@ -1,11 +1,14 @@
 package frc.robot.subsystems.roller
 
+import edu.wpi.first.wpilibj.I2C
 import frc.robot.lib.extensions.volts
 
 const val MOTOR_PORT = 0
 const val AUXILIARY_MOTOR_PORT = 1
 const val SENSOR_ID = 11
 
-val INTAKE = 1.volts
-val OUTTAKE = 1.volts
+val INTAKE = 12.volts
+val OUTTAKE = (-12).volts
```

Comment:
```suggestion
val OUTTAKE = -INTAKE
```
But isn't 12V too much?

Reply:
From our tests 12V is good.